src/test/scala/edu/ie3/mobsim/utils/IoUtilsSpec.scala

```diff
@@ -0,0 +1,205 @@
+        s"${secondEv.chargingAtHomePossible.toString}").replaceAll(",", ";")
+
+      list.get(list.size() - 2) shouldBe firstEvString
+      list.get(list.size() - 1) shouldBe secondEvString
+    }
+
+    "write electric vehicle charging stations correctly" in {
+      ioUtils.writeEvcs(
+        cs6,
+        currentlyAvailableChargingPoints,
+        currentTime,
+        uuid
+      )
+
+      val data = new BufferedReader(
+        new FileReader(new File(outputFileFolder + "evcs"))
+      )
+
+      val list: util.ArrayList[String] = new util.ArrayList[String]()
+      var line = data.readLine()
+
+      while (line != null) {
+        list.add(line)
+        line = data.readLine()
+      }
+
+      val chargingPoints: Int = cs6.chargingPoints
+      val occupiedChargingPoints: String =
+        (chargingPoints - availableChargingPoints.getOrElse(
+          cs6.uuid,
+          0
+        )).toString
+
+      val entry: String = (s"$uuid," +
+        s"$currentTime," +
+        s"${cs6.uuid}," +
+        s"$chargingPoints," +
+        s"$occupiedChargingPoints").replaceAll(",", ";")
+
+      list.get(list.size() - 1) shouldBe entry
+    }
+
+    "write pois correctly" in {
+      ioUtils.writePois(poiMap)
+
+      val data = new BufferedReader(
+        new FileReader(new File(outputFileFolder + "pois"))
+      )
+
+      val list: util.ArrayList[String] = new util.ArrayList[String]()
+      var line = data.readLine()
+
+      while (line != null) {
+        list.add(line)
+        line = data.readLine()
+      }
+
+      val string: String =
+        charging_hub_townPoi.nearestChargingStations.foreach {
+          case (evcsUuid, distance) =>
+            evcsUuid.toString + "," + distance.getValue.doubleValue().toString
+        }.toString
+
+      val compareString: String = (s"${charging_hub_townPoi.id}," +
+        s"${charging_hub_townPoi.getPoiType.toString}," +
+        s"${charging_hub_townPoi.size.toString}," +
+        s"$string").replaceAll(",", ";")
+
+      list.forEach { str =>
+        if (str.contains(uuid.toString)) {
+          str shouldBe compareString
+        }
+      }
+    }
+
+    "write positions correctly" in {
+      ioUtils.writeEvPosition(firstEv, currentTime, uuid)
+
+      val data = new BufferedReader(
+        new FileReader(new File(outputFileFolder + "positions"))
+      )
+
+      val list: util.ArrayList[String] = new util.ArrayList[String]()
+      var line = data.readLine()
+
+      while (line != null) {
+        list.add(line)
+        line = data.readLine()
+      }
+
+      val (location, destinationPoi) =
+        if (currentTime.isBefore(firstEv.parkingTimeStart)) {
+          ("DRIVING", "")
+        } else {
+          (
+            firstEv.destinationPoi.categoricalLocation.toString,
+            firstEv.destinationPoi.toString
+          )
+        }
+
+      val compareString: String = (s"$uuid," +
+        s"${firstEv.uuid.toString}," +
+        s"$location,").replaceAll(",", ";") +
```

Comment:
Why are you placing a `,` in the String and the replace it with `;`? Same also above

Reply:
Some information are seperated by ``,`` but need to be seperated by ``;``. Therefor I am using the replaceAll method.